fe/fe-core/src/main/java/com/starrocks/catalog/PartitionProperties.java

```diff
@@ -0,0 +1,97 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.catalog;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.Maps;
+import com.starrocks.analysis.PartitionDesc;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.FeConstants;
+import com.starrocks.common.FeNameFormat;
+import com.starrocks.common.util.PropertyAnalyzer;
+import com.starrocks.thrift.TTabletType;
+import org.apache.commons.lang.NotImplementedException;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class PartitionProperties extends PartitionDesc {
```

Comment:
I think PartitionProperties is better as a member of PartitionDesc, rather than inheritance.